### DISCUSSION TITLE
Track Usage

### DISCUSSION THREAD

**Discussion by byteinsight** [kutu/pyirsdk#38] [General]
I'm looking for Track Usage in percent. I can see a summary of track usage "SessionTrackRubberState": "high usage" but not the percentage I can find in the Garage61 exports. Its possible this value comes from the IBT files. 

I have asked on the iracing forum but the response from David was not definitive.  

Any chance someone can shed light on it?

Thanks :)

**Comment by kutu** [comment]
https://members.iracing.com/jforum/posts/list/2025/1470675.page#9450524

**Comment by byteinsight** [comment]
Legend.  Thank you.  The old forum was before my time and I didn't think to search it.   

Great software btw.   Thank you for developing it.
